src/router/index.ts

```diff
@@ -1,23 +1,44 @@
-import { createRouter, createWebHistory } from "vue-router"
-import HelloWorld from "../components/HelloWorld.vue"
+import { RouteRecordRaw, createRouter, createWebHistory } from "vue-router"
+import Home from "../views/Home.vue"
+import AboutMe from "../views/AboutMe.vue"
+import { nextTick } from "vue"
 
 /* For more info, visit https://router.vuejs.org/guide/ */
 
-const routes = [
+const routes: RouteRecordRaw[] = [
   {
-    // replace - this is your home route...
     path: "",
     name: "home",
-    component: HelloWorld,
+    component: Home,
     props: {
-      msg: "Vite + Vue",
-    },
+      msg: "Vite + Vue"
+    }
   },
+  {
+    path: "/about-me",
+    name: "about-me",
+    component: AboutMe,
+    meta: {
+      title: "About"
+    }
+  }
 ]
 
 const router = createRouter({
   history: createWebHistory(),
-  routes,
+  routes
+})
+
+//#region Title
+const DEFAULT_TITLE = "Daniel Ohrin"
+router.afterEach((to, _from) => {
```

Comment:
Why not have const title = "Daniel Ohrin"
and if to.meta.title === "string" then title +=